### ISSUE TITLE
Catalog update not showing completion because of session timeout

### ISSUE THREAD

**Issue by anettles** [ampache/ampache#463]
This has happed a couple of times now and I think it's because the session times out - mine is set to 7200 which I think makes it 2 hours.   When I hit Update All, I get the ajax_loader animation and no other indication of progress - which I know is another open issue.  So only after it's completely finished do I see the results of the catalog update process.

A couple of times I've come back to check on it (after plenty of time) and the ajax_loader no longer says loading and the window is still blank.  So I have no way of knowing if it finished or had errors.  Clicking anywhere won't show me the results either obviously and I end up having to login again of course.  Is this a separate issue than this one:

https://github.com/ampache/ampache/issues/404

 or should I increase my session timeout?
